### ISSUE TITLE
Ashwalker eggs are anchored while the Necropolis is loose

### ISSUE THREAD

**Issue by FantasticFwoosh** [tgstation/tgstation#40520]
**[No relevant round ID]**: can't remember, sorry. [will update this space when relevant.]
[Testmerges]: See above.

Ashwalker eggs cannot be moved from their fixed position upon spawning, which is annoying when they overlap, by comparison the Necropolis tendril centered around them can move off its static position, revealing a tile of actively dangerous lava it is normally centered upon sometimes being knocked back by carbon body-parts.

**Blaming merged Kevinz PR** #39556 for removing the anchored state.

``` cl
rscadd: All atom movables now have move force and move resist, and pull force An atom can only pull another atom if its pull force is stronger than that atom's move resist

rscadd: Mobs with a higher move force than an atom's move resist will automatically try to force the atom out of its way. This might not always work, depending on how snowflakey code is.

experimental: As of right now, everything has a move force and resist of 100, and a pull force of 101. Things take (resist - force) damage when bumped into

experimental; Failing to move onto a tile will now still bump up your last move timer. However, successfully pushing something out of your way will result in you automatically moving into where it was.
/cl
 ```

https://github.com/tgstation/tgstation/pull/39556#issuecomment-418154685 Kevinz was told about this but didn't do anything about it from the commit's i've read through looking for particular changes, with instead more broad implementations of their system [**disclaimer:** if i hadn't spotted it but it was never addressed]

- It has always been the case that the tendril could be moved by a office chair, and as such it is the only way to move it back when the necropolis breaks. It can also be moved off lavaland and to new locations doing this. Or dragging it outside so you can fulton it.

**Comment by FantasticFwoosh** [comment]
#40786 making a fix for this bug ( #40750 ) with a anchored state broke Travis, could we please consider reverting or prompting kevinz to make changes to #39556

[kevinz000 response](https://github.com/tgstation/tgstation/pull/40786#issuecomment-427957746)
>Making stuff respect move resist instead of anchoring them however I did cause the original bug with tendrils because somehow I caught mobs on throw pushing them but not items on throw 

- contributing.md indicts this is their mess to clean up with code maintenance and as much they admit that their PR broke this.

**Comment by kriskog** [comment]
Has been fixed
